android/MobiledgeXSDKDemo/README.md

```diff
@@ -16,80 +16,81 @@
 
 When you start the app, you will see your location on the map represented by a mobile phone icon. 
 
-In the real world, the location verification is implemented by the cellular network operator. For this demo, we use a simulator that stores location data, and is keyed on your current public IP address. To get an initial entry in the simulator database, tap the 3-dot menu, and select "Reset Location". Now the simulator considers your true location to be wherever your current GPS coordinates are. If your IP address stays the same, you won't need to repeat this step next time you run the app.
+In the real world, location verification is implemented by the cellular network operator. For this demo, we use a simulator that stores location data and is keyed on your current public IP address. To get an initial entry in the simulator database, tap the 3-dot menu, and select "Reset Location." Now the simulator considers the valid location to be wherever your current GPS coordinates are. If your IP address stays the same, you won't need to repeat this step next time you run the app.
 
 You can move the mobile phone icon by long-pressing on it and dragging it to a new location, or long-pressing anywhere on the map. When you release the icon, you will be presented with two options:
 - Spoof GPS at this location
 - Update location in GPS database.
 
-If you select "Spoof", this new location will be used for subsequent **Find Closest Cloudlet** and **Verify Location** calls. Depending on how close this location is to the location stored in the simulator, your next **Verify Location** call may return a different "accuracy" value, or return a failure. Tapping the mobile phone icon will the distance from the actual location, and any result code available.
+If you select "Spoof," this new location will be used for subsequent **Find Closest Cloudlet** and **Verify Location** calls. Depending on how close the location stored in the simulator is, your next **Verify Location** call may return a different "accuracy" value, or return a failure. Tapping the mobile phone icon will the distance from the actual location, and any result code available.
 
-If you select "Update location", this new location will be sent to the location simulator and will be used until another location is sent, or "Reset Location" is performed. This allows you to simulate having your real location be anywhere in the world, and you can verify that the expected cloudlet is found by **Find Closest Cloudlet**.
+If you select "Update location," this new location will be sent to the location simulator and will be used until another location is sent, or "Reset Location" is performed. Reset Location allows you to simulate your real location be anywhere in the world, and you can verify that **Find Closest Cloudlet** finds the expected cloudlet.
```

Comment:
"simulate your real location be anywhere" doesn't quite make sense. Maybe "being anywhere"?